### PR TITLE
Xresources's scrollrate messes with internal kscrollup/down calls

### PR DESCRIPTION
**Pull request by nikp123**
Anyway whenever kscrollup/down is called in the code, the scroll movement always gets limited to the value set by ``st.scrollrate set`` in the Xresources file. 
That means fxp. whenever I try to scroll up, but I need to type something out, the terminal moves down by ``st.scrollrate`` instead of just jumping down to the last line, like it regularly would.

NOTE: This patch considers that no one changed the default Arg.i = 1 in the config.h file and instead fixes the issue for users that use Xresources values, since in that case the kscrollup/down functions will get always called with Arg.i set to 1.

### CODE REVIEW

**Comment by actionless** [comment]
>  This patch considers that no one changed the default Arg.i = 1 in the config.h file and instead fixes the issue for users that use Xresources values, since in that case the kscrollup/down functions will get always called with Arg.i set to 1.

could you rephrase this somehow?

**Comment by nikp123** [comment]
> > This patch considers that no one changed the default Arg.i = 1 in the config.h file and instead fixes the issue for users that use Xresources values, since in that case the kscrollup/down functions will get always called with Arg.i set to 1.
> 
> could you rephrase this somehow?

In layman's terms: The patch ONLY works for people that dont change the scrollrate inside of config.h

**Comment by actionless** [comment]
it should check both, just prioritize the one from Xresources

**Comment by nikp123** [comment]
Sorry forgot that there was a literal ``scrollrate`` in config.

I meant that the default arguments for the kscrollup/down dont get changed in config.h
This: ``ShiftMask,            XK_Page_Up,     kscrollup,      {.i = -1} },``

But since I'm unable to test atm, please verify if the code behaves as described.

**Comment by actionless** [comment]
ah, you mean hotkeys

i've checked with the mouse -- seems to be fine, thanks!
